### ISSUE TITLE
q&v: query idea

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#2648]
Nothing we need immediately or even soon, but in writing up about the finds from last season's MKAP season it occurred to me that having a query that would allow one to see what was excavated in a specific season would be helpful, particularly if the same units are worked on over a span of multiple seasons. Maybe something that pulls up excavated materials from specific units per year? I imagine this would be useful for other projects, too.

**Comment by urapadmin** [comment]
see also #981

**Comment by urapadmin** [comment]
- if implemented as a query in Q&V this begs the same question as #3009: what should the content of the result list?

Wouldn't simply a time span instead of a season work? I could query all records (dayplans, loci, cm should do, I suspect) that were modified within a certain time span and show the identifiers so that one can jump to them.

**Comment by luizaogs** [comment]
Yes, time span works (season is perhaps more convenient, but I know things must not be developed based on user convenience!). Dayplans, loci, cm should be good. Maybe also unit?

**Comment by lbestock** [comment]
Units, please. Oh wow this will be helpful with Abydos already.
What’s the returned list going to look like? Looking at @luizaogs original request and thinking of how I will use this for Abydos, a unit-first list would be most useful. And then in fact I suspect CM will be most useful? Though the registration module in the recording app already does what I most need it to do there.

**Comment by urapadmin** [comment]
### Kiosk 1.6.27
- this query actually needs a new Kiosk version to be working
- note that for loci I am only checking the opening date, I hope that is sufficient (can't address the two different fields in a single record type. There is a ticket for that somewhere)

**Comment by luizaogs** [comment]
Which one of these is it? The top one has something weird going on in the description too. (They seem to say they do the same thing?)

<img width="463" alt="Image" src="https://github.com/user-attachments/assets/725b24cb-fbd3-49d7-bd21-447c49d7187e" />

**Comment by urapadmin** [comment]
the one with the strange description is the one I just added. I had actually forgotten that we already have the other query, which does virtually the same but spits out only units (while the other one spits out the exact identifier that has been altered).
So in my opinion the new query is better than the old one.

**Comment by urapadmin** [comment]
I uploaded the query again to ANC with a slight change of the title and fixed description:

![Image](https://github.com/user-attachments/assets/55a5e9da-1e58-4bd7-a5c1-74693b205e52)

**Comment by luizaogs** [comment]
Ok, it's working well. Did we already have the older query when I requested this? Huh. Sorry. 

Does it make sense to remove the older one? I guess if one only wants units it's still useful?

**Comment by urapadmin** [comment]
- more useful once I get to #2590

**Comment by luizaogs** [comment]
Agreed. So I close this.
